roles/cobbler/tasks/yum_systems.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install cobbler
+  yum:
+    name: "{{ cobbler_package }}"
+    state: latest
+  register: install_cobbler
+
+- name: Install syslinux
```

Comment:
I think these install tasks should also be in a list of packages to install and then used with 'with_items'.